lblod/frontend-mow-registry
# GN-4179: fix missing translations

## Overview
This PR fixes two translation issues:
- Usage of a wrong translation key for the search placeholder of related traffic lights
- Usage of a wrong translation key for the zonality of road-signs.

##### connected issues and PRs:
Fixes [GN-4179](https://binnenland.atlassian.net/browse/GN-4179?atlOrigin=eyJpIjoiMDlmYmUwZmM4ZjA5NDgxNjliZGIxOThiNDAzMzcxYTQiLCJwIjoiaiJ9)

### How to test/reproduce
- Visit a road-sign/road-marking detail page
- Add a related traffic-light
- Notice that the placeholder now has the correct translation key

- Visit a road-sign detail page
- Notice that the correct translation key is used if a road-sign has no explicit zonality specified (default: non-zonal)



### Checks PR readiness
